### DISCUSSION TITLE
由于墙的原因无法直接使用stream saver下载视频，需要开启魔法

### DISCUSSION THREAD

**Discussion by bixudamajiang** [the1812/Bilibili-Evolved#5076] [功能建议 / Ideas]
每次下b站视频却需要开代理会有点烦人，而且stream saver上一次更新都是在去年了
![image](https://github.com/user-attachments/assets/30625347-9402-485f-a37d-a452a87d22a3)

**Comment by the1812** [comment]
可以换 aria2
